pages/1.13/gui/secrets/index.md

```diff
@@ -10,6 +10,67 @@ model: /1.13/data.yml
 ---
 
 You can manage secrets and certificates from the Secrets page.
-![Secrets](/1.13/img/GUI-Secrets-Secrets_View_With_Secrets-1_12.png)
 
+![Secrets](/1.13/img/GUI-Secrets-Secrets_View_With_Secrets-1_12.png)
 Figure 1 - Secrets page
+
+
+For complete details on creating and managing Secrets, see the [Secrets](/1.13/security/ent/secrets) documentation.
+
+<p class="message--important"><strong>IMPORTANT: </strong>The maximum file size for a secret is approximately 1 MB, subtracting approximately 1 KB for the secret store metadata.</p>
+
+# Creating key-value pair secrets 
+
+1. Log in to the DC/OS UI as a user with the `dcos:superuser` permission.
```

Comment:
`dcos:superuser` is just as an example here, right? Any user with `dcos:secrets:default:* create` can create a secret.

@DominikDary can probably provide a better insight, but in general it is `dcos:secrets:default:[marathon-service-or-group-path] (create|read|update|delete)`

Reply:
We'll have to fix this later.